pkg/scanners/trivy/types.go

```diff
@@ -22,8 +18,23 @@ const (
 	StatusOK
 )
 
+const (
+	trivyCommandName        = "/trivy"
+	trivyJSONFormatFlag     = "--format=json"
+	trivyImageArg           = "image"
```

Comment:
nit: can we swap `trivyJSONFormatFlag` and `trivyImageArg` 😊

Reply:
done